### PR TITLE
Update ICSharpCode.Decompiler to 7.2.1.6856

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Fixes OOM when decompiling code with large byte[]s.

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -116,7 +116,7 @@
     <MicrosoftmacOSRefVersion>12.3.300-rc.3.83</MicrosoftmacOSRefVersion>
     <MicrosoftMetadataVisualizerVersion>1.0.0-beta3.21075.2</MicrosoftMetadataVisualizerVersion>
     <MicrosoftNETBuildExtensionsVersion>2.2.101</MicrosoftNETBuildExtensionsVersion>
-    <MicrosoftNETCorePlatformsVersion>2.1.2</MicrosoftNETCorePlatformsVersion>
+    <MicrosoftNETCorePlatformsVersion>5.0.0</MicrosoftNETCorePlatformsVersion>
```
Comment (CyrusNajmabadi) [inline eng/Versions.props:119]: @dotnet/roslyn-infrastructure any issues with this?  It was required for the new ICSharpCodeDecompiler version.  It also matches the line below which makes me feel like it's likely ok :D

Comment (arkalyanms) [reply eng/Versions.props:119]: LGTM! @dibarbet fyi to flag the insertion, if there are issues.

**Review by arkalyanms — APPROVED** [review]

**Review by dibarbet — APPROVED** [review]

**Comment by CyrusNajmabadi** [comment]
Tests are failing because of baselines:

image

File: src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenConstructorInitTests.cs [hunk 2]
```diff
@@ -1198,7 +1198,7 @@ class C
                 instance void .ctor () cil managed
         {
                 // Method begins at RVA 0x207f
-                // Code size 7 (0x7)
+                // Code size: 7 (0x7)
```
Comment (CyrusNajmabadi) [inline src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenConstructorInitTests.cs:1201]: Mechanical change.

**Comment by CyrusNajmabadi** [comment]
@dotnet/roslyn-compiler ptal.

**Comment by CyrusNajmabadi** [comment]
@jaredpar ptal at the compiler side.  thanks!

**Review by jcouv — APPROVED** [review]
Compiler changes (test-only) LGTM Thanks (iteration 7)

**Pull request merged**